queries/dune_v1/period_slippage.sql

```diff
@@ -139,14 +139,12 @@ incoming_and_outgoing as (
            transfer_type
     from batch_transfers i
              left outer join erc20.tokens t on i.token = t.contract_address
-    where
+    where tx_hash not in (select tx_hash from exluded_batches)
        -- We exclude settlements that have zero AMM interactions and settle several trades,
        -- as our query is not good enough to handle these cases accurately.
        -- Settlements with dex_swaps = 0 and num_trades = 0 can be handled in the following
-       -- and we want to consider them in order to filter out illegal behaviour
-        (dex_swaps = 0 and num_trades < 2) or dex_swaps > 0
-
-        and tx_hash not in (select tx_hash from exluded_batches)
+      -- and we want to consider them in order to filter out illegal behaviour
+      and ((dex_swaps = 0 and num_trades < 2) or dex_swaps > 0)
```

Comment:
This also appears to have been a bug in V1 which I find surprising because we never noticed. Also, the fact that I haven't yet updated the V1 query and it seems to be fine. Anyway, these are definitely different definitely statements as demonstrated [here](https://dune.com/queries/1656695). Note that this is a V1/Postgres query and the one linked in the later comment is a V2/Hive query --- and they do agree.

Reply:
The strangest things is that I was unable to get different results at all when fixing this test (that is, the results are identical with or without this fix). However, the boolean table linked in the previous comment does imply that this is a bug.